tests/test_instance.py

```diff
@@ -1,27 +1,46 @@
 import unittest
-from pynio import Instance
+from pynio import Instance, Block, Service
 from unittest.mock import MagicMock, patch
 from .mock import mock_service
 
 
+class MockInstance(Instance):
+
+    def __init__(self, host='127.0.0.1', port=8181, creds=None):
+        self._get_blocks = MagicMock()
+        self._get_blocks.return_value = {}, {}
+        self._get_services = MagicMock()
+        self._get_services.return_value = {}
+        super().__init__(host, port, creds)
+
+
 class TestInstance(unittest.TestCase):
-    def setUp(self):
-        blks = patch.object(Instance, '_get_blocks')
-        blks.start()
-        Instance._get_blocks.return_value = None, None
-        servs = patch.object(Instance, '_get_services')
-        servs.start()
 
     def test_instance(self):
-        i = Instance()
+        i = MockInstance()
         self.assertEqual(i.host, '127.0.0.1')
         self.assertEqual(i.port, 8181)
 
-    # TODO: Create a test that loads system settings
+    def test_add_block(self):
+        i = MockInstance()
+        i.blocks_types = MagicMock()
```

Comment:
Interesting, so this pretty much causes it to ignore all the setup for the template. i.blocks_types[...] just returns another MagicMock or something. I like how simple this makes it to just ensure that a block is added.